docs/source/backends-xnnpack.md

```diff
@@ -1,146 +1,69 @@
+ - `per_op_mode`: If true, emit individual delegate calls for every operator. This is an advanced option intended to reduce memory overhead in some contexts at the cost of a small amount of runtime overhead. Defaults to false.
+ - `verbose`: If true, print additional information during lowering.
 
+### Quantization
 
-[comment]: <> (TODO: Refactor quantizer to a more official quantization doc)
-## Quantization
-The XNNPACK delegate can also be used as a backend to execute symmetrically quantized models. For quantized model delegation, we quantize models using the `XNNPACKQuantizer`. `Quantizers` are backend specific, which means the `XNNPACKQuantizer` is configured to quantize models to leverage the quantized operators offered by the XNNPACK Library. We will not go over the details of how to implement your custom quantizer, you can follow the docs [here](https://pytorch.org/tutorials/prototype/pt2e_quantizer.html) to do so. However, we will provide a brief overview of how to quantize the model to leverage quantized execution of the XNNPACK delegate.
+Placeholder - document available quantization flows (PT2E + ao), schemes, and operators.
 
-### Configuring the XNNPACKQuantizer
+## Runtime Integration
 
-```python
-from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
-  XNNPACKQuantizer,
-  get_symmetric_quantization_config,
-)
-quantizer = XNNPACKQuantizer()
-quantizer.set_global(get_symmetric_quantization_config())
-```
-Here we initialize the `XNNPACKQuantizer` and set the quantization config to be symmetrically quantized. Symmetric quantization is when weights are symmetrically quantized with `qmin = -127` and `qmax = 127`, which forces the quantization zeropoints to be zero. `get_symmetric_quantization_config()` can be configured with the following arguments:
-* `is_per_channel`
-    * Weights are quantized across channels
-* `is_qat`
-    * Quantize aware training
-* `is_dynamic`
-    * Dynamic quantization
-
-We can then configure the `XNNPACKQuantizer` as we wish. We set the following configs below as an example:
-```python
-quantizer.set_global(quantization_config)
-    .set_object_type(torch.nn.Conv2d, quantization_config) # can configure by module type
-    .set_object_type(torch.nn.functional.linear, quantization_config) # or torch functional op typea
-    .set_module_name("foo.bar", quantization_config)  # or by module fully qualified name
-```
+The XNNPACK delegate is included by default in the published Android, iOS, and pip packages. When building from source, pass `-DEXECUTORCH_BUILD_XNNPACK=ON` when configuring the CMake build to compile the XNNPACK backend.
```

Comment:
This section really needs to be improved. Having three reference implementations:
-  CMake file for C++ 
-  iOS
-  Android